enterprise_access/settings/base.py

```diff
@@ -344,6 +347,9 @@ def root(*path_fragments):
         REQUESTS_LEARNER_ROLE,
         BFF_LEARNER_ROLE,
     ],
+    SYSTEM_ENTERPRISE_PROVISIONING_ADMIN_ROLE: [
+        PROVISIONING_ADMIN_ROLE,
+    ],
```

Comment:
The entire concept of adding a new system-wide role is new to me. Don't we have to create this on the edxapp side now?  Why not just assign PROVISIONING_ADMIN_ROLE to SYSTEM_ENTERPRISE_OPERATOR_ROLE so that only backend services and operational staff can call this endpoint?

Reply:
Good question - this role already exists (which surprised me). I'm not quite sure of the motivation for it, but it's there so I used it. But maybe we should additionally assign the provisioning feature role to the system operator role.

Reply:
> But maybe we should additionally assign the provisioning feature role to the system operator role.

Agreed! ➕ 1️⃣